test/compile/selection/layers.test.ts

```diff
@@ -85,16 +85,10 @@ describe('Layered Selections', () => {
               signal:
                 '"Horsepower: " + (format(datum["Horsepower"], "")) + "; Miles_per_Gallon: " + (format(datum["Miles_per_Gallon"], "")) + "; Origin: " + (isValid(datum["Origin"]) ? datum["Origin"] : ""+datum["Origin"])'
             },
-            fill: [
-              {
-                test: '!isValid(datum["Horsepower"]) || !isFinite(+datum["Horsepower"]) || !isValid(datum["Miles_per_Gallon"]) || !isFinite(+datum["Miles_per_Gallon"])',
-                value: null
-              },
-              {
-                scale: 'color',
-                field: 'Origin'
-              }
-            ],
+            fill: {
```

Comment:
FYI, intended change -- we no longer relying "null" fill to hide invalid marks in the new `"break-paths-keep-domains"` that replaces in the old secret "hide" mode. 

Same for the rest of this file.